dashboard/data/file_preprocessing/echo_files_parser.py

```diff
@@ -6,14 +6,74 @@
 class EchoFilesParser:
     def __init__(
         self,
-        echo_files_dir: str,
```

Comment:
What is the reason behind deleting thi one? Also, the test written for the class need to be modified, since without this attribute they won't pass.

Reply:
Since we will use EchoFilesParser not only to read from dir (moreover, reading from dir in the dashboard is not possible this way) so I think we shouldn't require this field in the creation of the Parser. Now simply depending on the way of reading files that is, their format, the appropriate method has to be chosen.

Reply:
Yeah there is one test that didn't work now and I will make it later

Reply:
Ok, I see now